src/core/OuterExamTimer.jsx

```diff
@@ -2,16 +2,15 @@ import React, { useEffect, useContext } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { AppContext } from '@edx/frontend-platform/react';
-import ExamStateContext from '../context';
-import { getLatestAttemptData } from '../data';
 import { ExamTimerBlock } from '../timer';
 import ExamAPIError from '../exam/ExamAPIError';
-import ExamStateProvider from './ExamStateProvider';
+import { getLatestAttemptData } from '../data/thunks';
```

Comment:
Nit. It seems like most components import from the top level `data` module (things exported from `data/index.js`). Should we follow that pattern here and in the other files?

Reply:
True, will update!